docs/time/index.rst

```diff
@@ -1132,10 +1132,19 @@ the location with which the |Time| object is initialized. Like the scale
 transformations, ERFA_ C-library routines are used under the hood, which support
 calculations following different IAU resolutions.
 
+Similarly, one can calculate the Earth rotation angle with
+:meth:`~astropy.time.Time.earth_rotation_angle`. Unlike sidereal time, which
+is referred to the equinox and is a complicated function of both UT1 and
+Terrestrial Time, the Earth rotation angle is referred to the Celestial
+Intermediate Origin (CIO) and is a linear function of UT1 alone.
+
+For the recent IAU precession models, as well as for the Earth rotation angle,
+the longitude includes the TIO locator rigorously corrected for polar motion.
```

Comment:
Define TIO and elaborate a bit on what it means to include the TIO locator.

Reply:
I'll give this a go.